waterfall/examples/simulator.rs

```diff
@@ -64,7 +66,7 @@ pub fn simulate(shape: Shape) {
         };
         let value = value.floor() as u64;
         if value != 0 {
-            heatmap.increment(Instant::now(), value, 1);
+            heatmap.increment(Instant::now(), value, 1).unwrap();
```

Comment:
When I run the example, this results in a panic. I think we may need to do `let _ = ...` to ignore the result of the increment. If that's not expected to error, then there's some bug.

Reply:
latest change addressed this with your suggestion as well as some config improvement.